src/App.vue

```diff
@@ -1,25 +1,33 @@
 <template>
   <div
-    v-if="config"
     id="app"
+    v-if="config"
     :class="[
-      `theme-${config.theme}`,
-      `page-${currentPage}`,
-      isDark ? 'dark' : 'light',
+      'theme-' + config.theme,
+      'page-' + currentPage,
+      isDark ? 'is-dark' : 'is-light',
```

Comment:
This is a breaking change, and it does not bring any value. Could you please revert this?